### PR TITLE
Loading of fake data

### PR DESCRIPTION
**Pull request by ChouSsu**
With regards to the discussion in #9862 (which addresses approach 2 of #9804) a less invasive change:

* moved 'load data' section to end of file
* added new needle to allow 3rd party extensions to add additional 'load columns' (```jhipster-needle-liquibase-add-loadcolumn```)

Open issue/topic: 
* Do we need a special treatment to disable foreign key constraint checks during load process (e.g.
```<sql dbms="h2">SET REFERENTIAL_INTEGRITY FALSE</sql>``` or 
```<sql dbms="mysql">SET FOREIGN_KEY_CHECKS = 0</sql>```
 (of course re-enable at end of load changeset))?

Fix: #9804 (utilizing slightly modified Approach 1)

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/react/package.json.ejs [hunk 1]
```diff
@@ -199,7 +199,7 @@ limitations under the License.
     "cleanup": "rimraf <%= DIST_DIR %>",
     <%_ if (protractorTests) { _%>
     "e2e": "protractor <%= TEST_SRC_DIR %>protractor.conf.js",
-    "postinstall": "node node_modules/webdriver-manager/bin/webdriver-manager update --gecko false",
+    "postinstall": "node node_modules/protractor/bin/webdriver-manager update --gecko false",
```
Comment (murdos) [inline generators/client/templates/react/package.json.ejs]: Could you rebase your branch onto master to remove these unrelated changes?

File: generators/server/needle-api/needle-server-liquibase.js [hunk 2]
```diff
@@ -48,6 +48,13 @@ module.exports = class extends needleServer {
         this.addBlockContentToFile(rewriteFileModel, errorMessage);
     }
 
+    addLoadColumnToEntityChangeSet(filePath, content) {
```
Comment (murdos) [inline generators/server/needle-api/needle-server-liquibase.js]: I would have designed this API differently, e.g. `addLoadColumnToEntityChangeSet(filePath, columnName, otherAttributes)` where otherAttributes whould be a map  and then would have filled a template `<column name="${columnName}" attribute1="..." attribute2="..." />` in the spirit of what is done for the maven needle api. 

But what you've done is consistent with the existing API of needle-server-liquibase

**Comment by ChouSsu** [comment]
@murdos I currently don't know what's going on - my rebase was looking good - but after a second refresh of the browser it's no longer looking that way, and my comments to your remarks are no longer available. Probably I've messed something up. Perhaps it's easier to start a new PR to avoid confusing git log, what do you think?

**Comment by murdos** [comment]
There's maybe an issue with your local `master` branch (maybe you made a rebase the wrong way), since I had no issue when I've rebased your branch locally. I've open https://github.com/jhipster/generator-jhipster/pull/9931

Either you try again cleaning things on your side, or we just merge https://github.com/jhipster/generator-jhipster/pull/9931. Just let me know 😉

**Comment by ChouSsu** [comment]
Finally got it cleaned up a bit.
Only got a "prettier" issue in the test file at the end - hence the 3rd commit.
